src/lib.rs

```diff
@@ -548,6 +558,14 @@ pub enum SubstepSet {
     ///
     /// See [`NarrowPhasePlugin`].
     NarrowPhase,
+    /// Responsible for running [`PostProcessCollisionsSchedule`] to allow user-defined systems
+    /// to post-process collisions, such as filtering.
```

Comment:
In "post-process collisions, such as filtering", the "such as" seems to be referring to the wrong thing (collisions instead of post-processing).

```suggestion
    /// Responsible for running the [`PostProcessCollisions`] schedule to allow user-defined systems
    /// to filter and modify collisions.
```

Here I assume that the schedule name was changed; if we land on something else, this will need to be changed of course

Reply:
Agree with your changes here.
I believe your assumption is correct. At the time I think I was trying to match your original wording for the comments, but I probably took it too far.